app/assets/javascripts/services/project_roles_svc.js

```diff
@@ -0,0 +1,28 @@
+samson.service('projectRolesService', function ($http) {
```

Comment:
Rubymine's default JS style to add spaces before parenthesis doesn't match the rest of our codebase.